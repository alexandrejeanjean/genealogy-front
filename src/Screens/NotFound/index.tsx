import React from "react";
import { Container, Row, Col } from "react-bootstrap";
import Main from "../Theme/index";

function PageNotFound() {
  return (
    <Main>
      <Container>
        <Row>
          <Col className="w-100 h-100 d-flex justify-content-center align-items-center">
            <p>Page not found... sorry</p>
          </Col>
        </Row>
      </Container>
    </Main>
  );
}

export default PageNotFound;
