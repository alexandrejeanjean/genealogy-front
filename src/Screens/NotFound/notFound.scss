.not-found-wrapper {
  background-color: #89c2d9;
  background: linear-gradient(259.07deg, #daf8ff 4.07%, #b2eaf8);
}
