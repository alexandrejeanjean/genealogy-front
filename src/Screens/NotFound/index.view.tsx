import React from "react";
import { Container, Row, Col } from "react-bootstrap";
import Main from "../Theme/index.view";
import { tree } from "../../assets/imgPath";
import "./notFound.scss";

const PageNotFound = () => {
  return (
    <Main>
      <Container fluid className="not-found-wrapper">
        <Row>
          <Col className="w-100 vh-100 d-flex justify-content-center align-items-center flex-column">
            <img src={tree} alt="" width="200px" />
            <p className="mt-3">Page not found... sorry</p>
          </Col>
        </Row>
      </Container>
    </Main>
  );
};

export default PageNotFound;
