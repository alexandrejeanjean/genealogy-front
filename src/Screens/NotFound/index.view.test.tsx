import React from "react";
import { shallow } from "enzyme";
import NotFound from "./index.view";

describe("NotFound", () => {
  it("should render correctly", () => {
    const Component = shallow(<NotFound />);
    expect(Component).toMatchSnapshot();
  });
});
