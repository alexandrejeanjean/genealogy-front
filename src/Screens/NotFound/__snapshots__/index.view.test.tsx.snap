// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<Component>
  <Container
    className="not-found-wrapper"
    fluid={true}
  >
    <Row
      noGutters={false}
    >
      <Col
        className="w-100 vh-100 d-flex justify-content-center align-items-center flex-column"
      >
        <img
          alt=""
          src={[Function]}
          width="200px"
        />
        <p
          className="mt-3"
        >
          Page not found... sorry
        </p>
      </Col>
    </Row>
  </Container>
</Component>
`;
